mlflow/utils/mlflow_tags.py

```diff
@@ -87,6 +87,9 @@
 # For automatic model checkpointing
 LATEST_CHECKPOINT_ARTIFACT_TAG_KEY = "mlflow.latest_checkpoint_artifact"
 
+# For prompt model configuration
+MLFLOW_PROMPT_MODEL_CONFIG = "mlflow.prompt.modelConfig"
```

Comment:
@chenmoneygithub Can we replace "." with "_" and move this config to `mlflow/prompt/constants.py`? Managed prompt registry does not allow to have "." in the tag key name.